nym-api/migrations/20240208120000_issued_credential_ecash.sql

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
+ * SPDX-License-Identifier: GPL-3.0-only
+ */
+
+
+DROP TABLE issued_credential;
+
+-- particular ecash credential issued in this epoch
+CREATE TABLE issued_credential
+(
+    id                         INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    epoch_id                   INTEGER NOT NULL,
+    tx_hash                    VARCHAR NOT NULL UNIQUE,
+    bs58_partial_credential    VARCHAR NOT NULL,
+    bs58_signature             VARCHAR NOT NULL,
+    joined_private_commitments VARCHAR NOT NULL,
+    expiration_date            INTEGER NOT NULL
```

Comment:
This ties up with the whole timestamp handling case. The underlying library uses `u64`, so I decided to handle `u64` as much as possible. We can debate about that, but ultimately, any fix will happen right before merging as it will affect the whole stack. 
TBD

Reply:
TO FIX : timestamps

Reply:
Fixed in 7bc7a8dde1e5c49778ba132e454a138dd1f0fdea